garage/tf/policies/gaussian_mlp_policy.py

```diff
@@ -4,16 +4,14 @@
 from garage.core import Serializable
 from garage.misc import logger
 from garage.misc.overrides import overrides
-from garage.tf.core import LayersPowered
-import garage.tf.core.layers as L
-from garage.tf.core.network import MLP
+from garage.tf.core import Parameterized
 from garage.tf.distributions import DiagonalGaussian
-from garage.tf.misc import tensor_utils
+from garage.tf.models import GaussianMLPModel
 from garage.tf.policies import StochasticPolicy
 from garage.tf.spaces import Box
 
 
-class GaussianMLPPolicy(StochasticPolicy, LayersPowered, Serializable):
+class GaussianMLPPolicy(StochasticPolicy, Parameterized, Serializable):
```

Comment:
* hmm i wonder if this has-a model or is-a model?

if it is-a model, then it gets `Parameterized` for free.

Reply:
This is has-a model

Reply:
haha i was asking what it *should be* not what it *is*

Reply:
@krzentner PTAL